package.json

```diff
@@ -19,12 +19,13 @@
     "clean": "rm -f vega-lite.* vega-lite-schema.json & find src -name '*.js*' -type f -delete & find test -name '*.js*' -type f -delete",
     "deploy": "npm run clean && npm run lint && npm run test && scripts/deploy.sh",
     "lint": "tslint -c tslint.json src/**/*.ts test/**/*.ts",
-    "start": "npm run watch:build & browser-sync start --server --files 'vega-lite.js' --index 'gallery.html'",
     "schema": "tsc && node src/schema/schemagen.js > vega-lite-schema.json",
+    "site": "bundle exec jekyll serve",
+    "start": "npm run watch & browser-sync start --server --files 'vega-lite.js' --index 'gallery.html'",
```

Comment:
Hmm, I'm not happy with this. If you use this and make a change in the code, it takes a long time before the gallery refreshes (10 seconds).